CMakeLists.txt

```diff
@@ -1,8 +1,42 @@
 cmake_minimum_required(VERSION 3.16)
-project(deepbacksub CXX)
+project(backscrub CXX)
 
+# allow override of Tensorflow location
+if(NOT DEFINED TENSORFLOW)
+  set(TENSORFLOW tensorflow)
+  set(TF_CHECKOUT true)
+endif()
+
+# find Git and use it to get backscrub version
+find_package(Git)
+if(GIT_FOUND)
+  execute_process(
+    COMMAND ${GIT_EXECUTABLE} describe --all --long --always --dirty
+       OUTPUT_STRIP_TRAILING_WHITESPACE
+       OUTPUT_VARIABLE DEEPSEG_VERSION)
```

Comment:
Consistent indentation depth of 2 per level?

Reply:
OK, have also fixed indentation on install target.